rdmo/management/rules.py

```diff
@@ -13,16 +15,19 @@ def is_editor(user) -> bool:
 
 
 @rules.predicate
-def is_editor_for_current_site(user, site) -> bool:
+def is_editor_for_current_site(user) -> bool:
     ''' Checks if any editor role exists for the user '''
     if not is_editor(user):
         return False  # if the user is not an editor, return False
-    return user.role.editor.filter(pk=site.pk).exists()
+    current_site = Site.objects.get_current()
```

Comment:
We could also use `settings.SITE_ID` and save a database hit, I guess. But thats not super important.